app/views/dashboard/admin/_navigation.html.slim

```diff
@@ -1,4 +1,6 @@
-nav.navbar.navbar-admin[role='navigation']
-  .navbar-collapse
-    ul.horizontal-menu
-      li= link_to 'Accessibility Statement', 'https://upaya.18f.gov/accessibility'
+.clearfix.p1
+  .sm-col-right
+    = link_to 'Accessibility Statement',
+      'https://upaya.18f.gov/accessibility', class: 'btn p1 h6 navy'
+    = link_to t('upaya.links.sign_out'),
+      destroy_user_session_path, method: 'delete', class: 'btn p1 h6 navy'
```

Comment:
Perhaps make this change in a separate PR or commit since it's not related to a/b testing?